examples/onnx/yolov5-videos/cortex.yaml

```diff
@@ -0,0 +1,13 @@
+# WARNING: you are on the master branch, please refer to the examples on the branch that matches your `cortex version`
+
+- name: youtube-yolov5
+  kind: SyncAPI
+  predictor:
+    type: onnx
+    path: predictor.py
+    model_path: yolov5s.onnx
+    config:
+      iou_threshold: 0.5
+      confidence_threshold: 0.3
+  compute:
+    gpu: 1
```

Comment:
Consider adding a comment on this line, e.g. `# This is optional, since the API will run on CPUs (albeit slower)`